playset/session.py

```diff
@@ -0,0 +1,214 @@
+import asyncio
+import random
+from collections import Counter
+from redbot.core.utils.chat_formatting import box
+from redbot.core.data_manager import cog_data_path
+import discord
+import numpy as np
+import matplotlib.pyplot as pp
+import os
+import wget
+from zipfile import ZipFile
+
+__all__ = ["SetSession"]
+
+_CARD_SIZE = (84,61)
+_VALID_TRIPLES = [(0,0,0),(1,1,1),(2,2,2),(0,1,2),(0,2,1),(1,0,2),(1,2,0),(2,0,1),(2,1,0)]
+_LETTER_MAP = {"A":(0,0),"B":(0,1),"C":(0,2),"D":(0,3),"E":(0,4),"F":(0,5),"G":(0,6),
+               "H":(1,0),"I":(1,1),"J":(1,2),"K":(1,3),"L":(1,4),"M":(1,5),"N":(1,6),
+               "O":(2,0),"P":(2,1),"R":(2,2),"S":(2,3),"T":(2,4),"U":(2,5),"V":(2,6)}
+               
+
+
+class SetSession:
+    def __init__(self, ctx):
+        self.dataDir = str(cog_data_path(self))
+        if not 'cards' in self.dataDir:
```

Comment:
It would also be nice if there were some mechanism to determine whether the files are out of date. (You don't have to actually use it right now but retaining a timestamp or hash could be useful later.)

Reply:
forgot an os.listdir. Will fix. And good point on the versioning. Maybe a force update command will do for now.